logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/webSocketStatus/EnableWebSocketUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.user.webSocketStatus
+
+import com.wire.kalium.logic.StorageFailure
+import com.wire.kalium.logic.configuration.UserConfigRepository
+import com.wire.kalium.logic.functional.Either
+
+interface EnableWebSocketUseCase {
+    operator fun invoke(enabled: Boolean): Either<StorageFailure, Unit>
+}
+
+class EnableWebSocketUseCaseImpl(
+    private val userConfigRepository: UserConfigRepository
+) : EnableWebSocketUseCase {
+    override operator fun invoke(enabled: Boolean) = userConfigRepository.persistWebSocketStatus(enabled)
+}
```

Comment:
If we have its name as `Enable` then we should not receive a boolean and just assume we are passing a boolean true to `persistWebSocketStatus`, no? 🤔

Reply:
hmmm  , you have a point there , I think it's better to go with the name persist web socket  status